cores/epoxy/Esp.h

```diff
@@ -15,6 +15,8 @@ class EspClass
 
 		void reset() {};
 
+		void restart() {};
```

Comment:
Technically the trailing `;` is not necessary there, but you are just copying the `reset() {};` above it, so thank you for being consistent. I'll fix this with a subsequent commit.